app/page.tsx

```diff
@@ -19,6 +28,10 @@ const Home = () => {
       songName="SOS!"
       rank="12"
       />
+      <div>
+      <Search 
+      onChange={handleInputChange}/>
```

Comment:
u don need external function directly pass setQuery and remove handleInputChange